actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -16,8 +16,22 @@ module Rendering
     extend ActiveSupport::Concern
     include ActionView::ViewPaths
 
-    # Normalizes arguments, options and then delegates render_to_body and
+    # Normalizes arguments, options and then delegates <tt>render_to_body</tt> and
     # sticks the result in <tt>self.response_body</tt>.
+    #
+    # The primary options are:
```

Comment:
You bet. Can you tell if this should only be part of `rendering.rb` or should this also be part of the other file, `rendering_helper.rb` as well? I don't have a clear sense if both files should be identical. Thanks.